portfolio/src/main/webapp/index.html

```diff
@@ -6,18 +6,19 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <img id="pic" src="./images/nick.jpg" />
+  <body id = "background">
     <div id="content">
-      <h1>Nick's Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>My name is Nicholas Lumiere. I'm 19 years old and I live in Seattle,
+        <img id="pic" src="./images/nick.jpg" />
+        <h1>Nick's Portfolio</h1>
+        <p>My name is Nicholas Lumiere. I'm 19 years old and I live in Seattle,
            WA. I study computer science at CU Boulder. I am a Google STEP intern
             (quarantine edition) which is why this page exists. </p>
-      <p>Click here to get a random message:</p>
+        <button class = "button" onclick="visitLink('https://www.linkedin.com/in/nicholas-lumiere-926867187/')">LinkedIn</button>
+        <button class = "button" onclick="visitLink('https://www.github.com/nlumiere/')">GitHub</button>
```

Comment:
If these are made as anchors, you can use href= to link them to other pages without needing to create a helper method in javascript.
(i.e. <a href="https://www.linkedin.com/in/nicholas-lumiere-926867187/">LinkedIn</a>)
In my opinion linked text looks more like it will take you to somewhere else than a button does as a minor UX note.